### DISCUSSION TITLE
How to make iptable rule such that only required ports are redirected to proxy

### DISCUSSION THREAD

**Discussion by tigllon** [istio/istio#51763] [General]
Hi Folks !!
My used case is I want to create such iptables rule so that only mentioned ports are redirected to proxy rest are not. 
Regarding this, I tried to use the  -q option of [istio-iptables](https://istio.io/latest/docs/reference/commands/pilot-agent/#pilot-agent-istio-iptables), but the issue with this option is, it not only redirects the ports given to it but also the others.
So for example if I give -q list of ports like {80,81,82} , it will redirect the request coming to these ports to proxy. But ,if I made request to port 83, that will also be redirect, to proxy and this should not happen, if I am correct.
Can someone help me in this ?

PS-: the image which I am using is istio/proxyv2:1.12.0

**Comment by zirain** [comment]
will https://istio.io/latest/docs/reference/config/annotations/#SidecarTrafficIncludeInboundPorts help?
